packages/web-components/src/toolbar/toolbar.styles.ts

```diff
@@ -0,0 +1,97 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusMedium,
+  colorNeutralBackground1,
+  colorNeutralForeground1,
+  fontFamilyBase,
+  spacingHorizontalNone,
+  spacingHorizontalS,
+  spacingHorizontalXL,
+  spacingHorizontalXS,
+} from '../theme/design-tokens.js';
+
+/**
+ * Toolbar styles
+ * @public
+ */
+export const styles = css`
```

Comment:
### Button Background

The pressed (`:hover:active`) state of the toolbar buttons should be have a darker background than the toolbar background per the Figma spec:

<img width="802" alt="image" src="https://github.com/microsoft/fluentui/assets/5067638/16dfc504-f5df-43bf-92ff-cdb8c9e118c0">

Implementation:

<img width="626" alt="image" src="https://github.com/microsoft/fluentui/assets/5067638/3791cf47-26b5-4f8c-8780-f9cbe5df47e2">

### Button Color

The button text color should also remain near-white on :hover:active, but it is changing to brand.

Reply:
Addressed in new PR